recipe/meta.yaml

```diff
@@ -1,48 +1,38 @@
-{% set version = "0.10" %}
+{% set version = "1.0.2" %}
 
 package:
   name: libblitz
   version: {{ version }}
 
 source:
-  fn: blitz-{{ version }}.tar.gz
-  url: http://downloads.sourceforge.net/project/blitz/blitz/Blitz%2B%2B%20{{ version }}/blitz-{{ version }}.tar.gz
-  md5: 66268b92bda923735f2e3afc87dcb58a
+  git_url: https://github.com/blitzpp/blitz.git
+  git_rev: {{ version }}
 
 build:
-  number: 2
-  skip: true  # [win and py27]
-  features:
-    - vc9   # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py>=35]
+  number: 0
 
 requirements:
   build:
-    - python  # [win]
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py>=35]
-  run:
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py>=35]
+    - python
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
 
 test:
   commands:
     - test -d $PREFIX/include/blitz        # [unix]
     - test -e $PREFIX/lib/libblitz.a       # [unix]
     - test -e $PREFIX/lib/libblitz.so      # [linux]
     - test -e $PREFIX/lib/libblitz.dylib   # [osx]
-    - if not exist %PREFIX%\Library\bin\blitz.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\blitz.lib exit 1  # [win]
```

Comment:
```suggestion
```
I don't know much about library compiling on windows, but I see you removed the parts in `bld.bat` that copied the `.dll` and `.lib` files, but kept the check for the `.lib` file here.

Reply:
I was trying to replicate these steps: https://github.com/blitzpp/blitz/blob/1.0.2/appveyor.yml but looks like cmake does not copy them.